README.md

```diff
@@ -9,14 +9,15 @@ Both are created, run and managed by ReDI School of Digital Integration. We're a
 
 ## Getting started for developers
 
-1. Make sure mongodb is installed your computer
-2. If you're re-seeding, drop the database files: `rm -rf mongodb-data/*`
-3. Run `yarn` in project root to install dependencies
-4. Run `yarn start:all` to boot all apps, _or_ a subset of apps using the `start:x` commands in package.json.
+1. Make sure [MongoDB](https://docs.mongodb.com/manual/installation/) is installed on your computer.
+2. If you're re-seeding, drop the database files: `rm -rf mongodb-data/*` and run `yarn seed` to seed the database.
```

Comment:
Can we find a nice way to specify _when_ this is actually needed? Since we might have some people running MongoDB in "service" mode, where it's already running "outside" the repository and the data files won't be stored in this folder.